src/testdir/test_popup.vim

```diff
@@ -896,4 +896,48 @@ func Test_menu_only_exists_in_terminal()
   endtry
 endfunc
 
+func Test_popup_complete_mode()
+  new
+  inoremap <f5> <c-r>=complete_mode()<cr>
+  call writefile([
+        \ '!rm	src/testdir/amiga.vim	/^cmap !rm !Delete all$/;"	m',
+        \], 'dummy.txt')
```

Comment:
In general, temporary files created by Vim tests start with X presumably because it makes it easier to spot temporary files which were not removed.